apps/web/src/pages/overview/components/Staking.vue

```diff
@@ -138,34 +139,42 @@ watch(formattedAmountToStake, async () => {
   }
 })
 
-watch(user, () => {
-  aggregateAddressesByProvider()
-  termsOfServiceCheckbox.value = user.value?.agreedToTermsOfService as boolean
+watch(user, async () => {
+  if (user.value?.id) {
+    aggregateAddressesByProvider()
+    termsOfServiceCheckbox.value = user.value?.agreedToTermsOfService as boolean
+    address_balance.value = (Math.round( await getEthersBalance(user.value?.address as string) * 100) / 100 ) + ' ETH'
+    selectedWalletAddress.value = user.value?.address as string
+    selectedStakingProvider.value = user.value?.walletProvider as ProviderString
+  } else {
+    selectedStakingProvider.value = ''
+    selectedWalletAddress.value = null
+    formattedAmountToStake.value = ''
+    address_balance.value = null
+  }
 })
 
 onMounted(async () => {
   window.addEventListener('click', handleOutsideClick)
   aggregateAddressesByProvider()
   currentEthPrice.value = Math.round((await getCurrentPrice({coin: 'ETH', currency: 'USD'})) * 100) / 100
   estimatedFees.value = await getDepositFees()
+  if (user.value?.id) {
+    address_balance.value = (Math.round( await getEthersBalance(user.value?.address as string) * 100) / 100 ) + ' ETH'
+    selectedStakingProvider.value = user.value?.walletProvider as ProviderString
+  }
+  selectedWalletAddress.value = user.value?.address as string
 })
 
-
-onUnmounted(() =>{
+onUnmounted(() => {
   window.removeEventListener('click', handleOutsideClick)
 })
 
-const stakingActionLoader = ref(false)
-const loading = ref(false)
```

Comment:
these variables were necessary for the front-end animation, can we change these back please, or adjust the front-end to work like it did without them?

Reply:
I don't remember removing them. Must have been an accident (although a reference error should have thrown if I didn't have them defined but left the use of them in there)

Reply:
Oh. I didn't remove them. I just moved them up to declare all of the variables together. You can move em back I guess unless @shanejearley has a preference.